Dockerfile

```diff
@@ -1,19 +1,32 @@
-FROM debian:stable
+FROM python:3.6-jessie
```

Comment:
What about starting from [3.6-alpine or 3.6-slim-stretch](https://hub.docker.com/_/python/) instead? They will be smaller footprints.

Reply:
Actually, slim-stretch will pull from debian repos, so that will be the same repos that raspian pulls from for the rover.

Reply:
Done.